pkg/email/send_email.go

```diff
@@ -75,3 +77,13 @@ func (e *Emailer) SendInviteUserEmail(receivers []string, params map[string]stri
 	}
 	return e.SendEmailWithLimit(receivers, inviteUserWEmailSubject, body, nil)
 }
+
+func (e *Emailer) IsEmailerConfigured() bool {
```

Comment:
In this case, cache the state of the email as part of the parameters is not a good practice - you don't know when it is updated as it can be updated externally.
- Name: IsEmailerConfigured -> IsConfigured() or IsEnabled(), will remove the name of the entity from query the state
- Just return the result as recalling it doesn't cache the state and we don't need this cache at this point.

Reply:
Fixed